imx/signers/stark/signer.go

```diff
@@ -84,7 +84,7 @@ func (base *Signer) VerifySignature(hash *big.Int, signature, signersPublicKey s
 }
 
 // GetAddress returns the stark public key of the StarkSigner as a 64 digit hex string prefixed with 0x.
```

Comment:
Are we not keeping these method names in sync with the core-sdk-ts?

Reply:
Also, if this method name is changed it should also change on the godoc comment above.

Reply:
Yup, godoc needs update. Typescript SDK will need to be updated as well.

Reply:
Created https://immutable.atlassian.net/browse/DX-1310 to address Core SDK TS